examples/daemonsetsharding/deployment-no-node-pods.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: exporter
+    app.kubernetes.io/name: kube-state-metrics
+    app.kubernetes.io/version: 2.10.0
+  name: kube-state-metrics
```

Comment:
nit: kube-state-metrics -> kube-state-metrics-global

We have another deployment with the same name. In the future, we can let ` kube-state-metrics-global` to collect all other metrics which can't be sharded by node.